From 5e8a9afc0e83326be48295e5fd20cffe73aa9b1e Mon Sep 17 00:00:00 2001
From: Ralph Bean <rbean@redhat.com>
Date: Tue, 26 Jun 2012 13:55:24 -0400
Subject: [PATCH] Appropriate GITFLOW_DIR.

---
 git-flow |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/git-flow b/git-flow
index 93e9f0f..7a7d67f 100755
--- a/git-flow
+++ b/git-flow
@@ -45,9 +45,7 @@ if [ "$DEBUG" = "yes" ]; then
 	set -x
 fi
 
-# The sed expression here replaces all backslashes by forward slashes.
-# This helps our Windows users, while not bothering our Unix users.
-export GITFLOW_DIR=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
+export GITFLOW_DIR="/usr/share/gitflow"
 
 usage() {
 	echo "usage: git flow <subcommand>"
-- 
1.7.10.4

